contrib/knot/template.py

```diff
@@ -0,0 +1,54 @@
+pkgname = "knot"
+pkgver = "3.3.8"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-dnstap",
+    "--enable-fastparser",
```

Comment:
you also want --enable-quic, and then nghttp2+ngtcp2 in makedeps (i'll add the latter right now)

Reply:
considering that ngtcp2 isn't packaged yet, I'd like to push that out for a follow-up

Reply:
it is

Reply:
(i said 'right now' and added it 10 minutes later :) )

Reply:
indeed.

Reply:
I apparently missed reading the bit in brackets...